src/Swarm/Language/Syntax.hs

```diff
@@ -426,79 +433,131 @@ isUserFunc c = case constMeta $ constInfo c of
-  AppF -> binaryOp "$" 0 R
-  Teleport -> commandLow 2
-  As -> commandLow 2
-  RobotNamed -> commandLow 1
-  RobotNumbered -> commandLow 1
-  Knows -> commandLow 1
+  Wait -> commandLow 0 "Wait for a number of time steps."
+  Noop ->
+    commandLow 0 . doc "Do nothing." $
+      [ "This is different than 'Wait' in that it does not take up a time step."
+      , "It is useful for commands like if, which requires you to provide both branches."
+      , "Usually it is automatically inserted where needed, so you do not have to worry about it."
+      ]
+  Selfdestruct ->
+    commandLow 0 . doc "Self-destruct the robot." $
+      [ "Useful to not clutter the world."
+      , "This destroys the robots inventory, so consider 'salvage' as an alternative."
+      ]
+  Move -> commandLow 0 "Move forward one step."
+  Turn -> commandLow 1 "Turn in some direction."
+  Grab -> commandLow 0 "Grab an item from the current location."
+  Harvest ->
+    commandLow 0 . doc "Harvest an item from the current location." $
+      [ "Leaves behind a growing seed if the harvested item is growable."
+      , "Otherwise it works exactly like 'grab'"
+      ]
+  Place ->
+    commandLow 1 . doc "Place an item at the current location." $
+      ["The current location has to be empty for this to work."]
+  Give -> commandLow 2 "Give an item to another robot closeby."
+  Install -> commandLow 2 "Install a device from inventory on a robot."
+  Make -> commandLow 1 "Make an item using a recipe."
+  Has -> commandLow 1 "Sense whether the robot has a given item in its inventory."
+  Count -> commandLow 1 "Get the count of a given item in robots inventory."
+  Reprogram ->
+    commandLow 2 . doc "Reprogram another robot with new command." $
+      ["The other robot has to be closeby and idle."]
+  Drill ->
+    commandLow 1 . doc "Drill through an entity." $
+      [ "Usually you want to 'drill forward' when exploring to clear out obstacles."
+      , "When you have found a source to drill, you can stand on it and 'drill down'."
+      , "See what recipes with drill you have available."
+      ]
+  Build ->
+    commandLow 1 . doc "Construct a new robot." $
+      [ "You can specify a command for the robot to execute."
+      , "If the command requires a device it will be installed from your inventory."
+      ]
+  Salvage ->
+    commandLow 0 . doc "Deconstruct an old robot." $
+      ["Salvaging a robot will give you its inventory, installed devices and log."]
+  Say ->
+    commandLow 1 . doc "Emit a message." $ -- TODO: #513
+      [ "The message will be in global log, which you can not currently view."
+      , "https://github.com/swarm-game/swarm/issues/513"
+      ]
+  Log -> commandLow 1 "Log the string in the robots logger."
+  View -> commandLow 1 "View the given robot."
+  Appear ->
+    commandLow 1 . doc "Set how the robot is displayed." $
+      [ "You can either specify one character or five (for each direction)."
+      , "The default is \"X^>v<\"."
```

Comment:
Because the robot is looking down. So it looks like four robots meet.

Anyway, I would like to make that customizable one day, so that powerline font users can choose prettier versions.

Reply:
I understand, but how would the player ever be able to see that?

Reply:
You mean `build {move; turn down}`?

Reply:
\*gape\* Wow, I never knew that a robot cold turn downwards!